msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2582290.java

```diff
@@ -0,0 +1,160 @@
+        msalTestApp.handleFirstRun();
+
+        // Click on "AcquireToken" button
+        // User is Prompted for creds
+        final MicrosoftStsPromptHandlerParameters promptHandlerParameters = MicrosoftStsPromptHandlerParameters.builder()
+                .prompt(PromptParameter.SELECT_ACCOUNT)
+                .loginHint(username)
+                .sessionExpected(false)
+                .broker(mBroker)
+                .expectingBrokerAccountChooserActivity(false)
+                .expectingProvidedAccountInBroker(false)
+                .expectingLoginPageAccountPicker(false)
+                .expectingProvidedAccountInCookie(false)
+                .consentPageExpected(false)
+                .passwordPageExpected(true)
+                .speedBumpExpected(false)
+                .registerPageExpected(false)
+                .enrollPageExpected(false)
+                .staySignedInPageExpected(false)
+                .verifyYourIdentityPageExpected(false)
+                .howWouldYouLikeToSignInExpected(false)
+                .build();
+
+        // Enter username and pwd from step 2 to complete the acquire token flow
+        // Token should be retrieved succeffully
+        final String token = msalTestApp.acquireToken(username, password, promptHandlerParameters, true);
+        Assert.assertNotNull(token);
+        msalTestApp.handleBackButton();
+
+        // Click on "Get Users" button
+        // The user account should be shown in the UI
+        final List<String> users = msalTestApp.getUsers();
+        Assert.assertTrue(users.size() == 1);
+        Assert.assertTrue(users.get(0).contains(username));
+        msalTestApp.handleBackButton();
+
+        // Click on "Acquire Token Silent" button
+        // Token should be retrieved successfully
+        final String silentToken = msalTestApp.acquireTokenSilent();
+        Assert.assertNotNull(silentToken);
+        msalTestApp.handleBackButton();
+
+        // Select the Auth scheme as "POP"
+        msalTestApp.selectFromAuthScheme("POP");
+
+        // Click on "Generate SHR" button
+        // UI should be updated with an SHR token
+        final String shrToken = msalTestApp.generateSHR();
+        Assert.assertNotNull(shrToken);
+        msalTestApp.handleBackButton();
+
+        // Click on "Remove User" button
+        // UI updated with message "The account is successfully removed"
+        final String removeUserMessage = msalTestApp.removeUser();
+        Assert.assertEquals("The account is successfully removed.", removeUserMessage);
+
+        // Install updated oneAuthTestApp
+        final OneAuthTestApp oneAuthTestApp = new OneAuthTestApp();
+        oneAuthTestApp.install();
+        oneAuthTestApp.launch();
+        oneAuthTestApp.handleFirstRun();
+
+        // Enter username in account name
+        oneAuthTestApp.handleUserNameInput(username);
+        oneAuthTestApp.selectFromAppConfiguration("com.microsoft.identity.LabsApi.Guest");
```

Comment:
as this is a standard step across all the OneAuthTestApp testcases, can we do this as part of handleFirstRun?

Reply:
I suppose we will do these steps only when we wanna sign in without handling the prompt?  I'm not adding them before the regular sign in process. Do we also need to perform them before regular sign in process?

Reply:
Are you able to do a regular sign-in to OneAuth app without using this config? I believe this config should be used whenever broker app is installed and user tries to sign-in using OneAUthTest App

Reply:
Added these two steps into handleFirstRun in OneAuthTestApp infra and remove lines in the corresponding test cases.